spec/support/shipment_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class ShipmentDecorator < SimpleDelegator
+class ShipmentDecoratorStub < SimpleDelegator
```

Comment:
I renamed it because I had a conflict when use with Aday, which also has a class named `ShipmentDecorator`, probably in the same namespace.
I don't mind reverting, but it means i will have to change it in ADAY then.